backend/danswer/connectors/google_drive/connector.py

```diff
@@ -45,7 +47,10 @@ def get_folder_id(
     """
     Get the ID of a folder given its name and the ID of its parent folder.
     """
-    query = f"'{parent_id}' in parents and name='{folder_name}' and mimeType='{DRIVE_FOLDER_TYPE}'"
+    query = (
+        f"'{parent_id}' in parents and name='{folder_name}' and "
```

Comment:
Line longer than black's 88 but it doesn't fix long strings.

Reply:
Will soon add github actions to auto format and detect type check issues to make things easier for contributors